docs/docs/deploy/baremetal.md

```diff
@@ -560,7 +560,7 @@ pm2 monit
 
 ![pm2 dashboard](https://user-images.githubusercontent.com/300/164799386-84442fa3-8e68-4cc6-9e64-928b8e32731a.png)
 
-And even a web-based UI with paid upgrades if you need to give normies access to your monitoring data:
```

Comment:
I think "normies" here is an intentional pun.

Reply:
rebased... per your feedback. 

OTOH imho this kind of slang should be avoided. 
- it makes things harder for non native (US bound) english readers (i had to check - and i have a _reasonable_ understanding of english)
- makes translating harder. (as translations stop being 1-1)  

All the best, 

António